internal/shellgen/flake_input.go

```diff
@@ -67,10 +67,12 @@ func (f *flakeInput) BuildInputs() []string {
 func flakeInputs(ctx context.Context, devbox devboxer) ([]*flakeInput, error) {
 	defer trace.StartRegion(ctx, "flakeInputs").End()
 
-	inputs := map[string]*flakeInput{}
+	// Use the verbose name flakeInputs to distinguish from `inputs`
+	// which refer to `nix.Input` in most of the codebase.
+	flakeInputs := map[string]*flakeInput{}
 
-	userPackages := devbox.PackagesAsInputs()
-	pluginPackages, err := devbox.PluginManager().PluginPackages(userPackages)
+	userInputs := devbox.PackagesAsInputs()
```

Comment:
I think this is regressing. Moving towards naming things packages will be easier to understand imo

Reply:
From my PR summary:

> nomenclature: I have tried to ensure consistency around:
- inputs referring to nix.Input
- flakeInputs referring to shellgen.flakeInput
- packages referring to raw strings that are in the packages list in devbox.json.